examples/hotel-bookings/README.md

```diff
@@ -0,0 +1,36 @@
+[Framework examples →](../)
+
+# Hotel bookings
+
+View live: <https://observablehq.observablehq.cloud/framework-example-hotel-bookings/>
+
+This dashboard explores reservation data for a resort in Portugal from Antonio _et al._ (2021), recorded between July 2015 and August 2017 to highlight differences in daily rates, visit seasons, guest nationality, and room type reserved for different booking types (_e.g._ corporate, direct, or online reservations).
+
+**Data source:** Antonio et al (2021). Hotel booking demand datasets. Data in Brief (22): 41-49. https://doi.org/10.1016/j.dib.2018.11.126
+
+## Implementation
+
+```
+.
+├── README.md
+├── observablehq.config.js
+├── package.json
+└── src
+    ├── components
+    │   ├── bigNumber.js
+    │   └── donutChart.js
+    ├── data
+    │   ├── hotelData.csv.js
+    │   └── hotels.csv
```

Comment:
needs update

Reply:
Done! (also updated the rest of the README to reflect recent changes)